src/main/drivers/timer.c

```diff
@@ -558,6 +558,8 @@ void timerChICPolarity(const timerHardware_t *timHw, bool polarityRising)
     timHw->tim->CCER = tmpccer;
 }
 
+// XXX timerChCCRHi and timerChCCRLo not used?
+#if 0
```

Comment:
These are my work. It was commented out during fixing the `timerChCCR` (although it still has a bug), because I couldn't understand the logic behind the operation using channel 2 mask.

Reply:
It is possible to capture rising and falling edge separately (one pin) when using timer pair. It was useful for fast softserial, but only timer part got merged. Other code still supports this functionality